src/chat/chatter.py

```diff
@@ -295,11 +296,13 @@ def pressed(self, item):
                 action_inspect_in_mordor = QtGui.QAction("Send the Orcs", menu)
                 menu.addAction(action_inspect_in_mordor)
 
+                ROOT = Settings.get('mordor/host')
+
                 def send_the_orcs():
                     if self.id != -1:
-                        QtGui.QDesktopServices.openUrl(QUrl("http://mordor.faforever.com/users/{}".format(self.id)))
+                        QtGui.QDesktopServices.openUrl(QUrl("http://" + ROOT + "/users/{}".format(self.id)))
```

Comment:
Oh cool, that works? Probably better do that then! This will work for all the instances I've changed in this PR?